src/main/js/controls/Gantt/helpers/translateHelpers.js

```diff
@@ -405,12 +433,15 @@ const translateDataPoints = (scale, config, trackPath) => {
  * @private
  * @param {object} scale - d3 scale for Graph
  * @param {Selection} path - d3 object of track
+ * @param {object} graphConfig - graph config needed for panning feature
  * @returns {undefined} - returns nothing
  */
-const translateTaskBar = (scale, path) =>
+const translateTaskBar = (scale, path, graphConfig) =>
```

Comment:
`config`. Let's not confuse this with Graph API's config.

Reply:
I will correct it